apps/admin/src/app/users/[id]/page.tsx

```
@@ -0,0 +1,204 @@
+"use client";
+
+import { useParams } from "next/navigation";
+import Link from "next/link";
+import Image from "next/image";
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from "@buzztrip/components";
+import { Badge, Button } from "@buzztrip/components/ui";
+import { useConvexUserById } from "@/hooks/use-admin-data";
+import { ArrowLeft, Calendar, Users, Map, Eye } from "lucide-react";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)